RockWeb/Themes/Rock/Layouts/Site.Master

```diff
@@ -1,189 +1,189 @@
-			                </div>
-                    </nav>
-
-                </header>
-
-                <main>
-                    <asp:ContentPlaceHolder ID="feature" runat="server"></asp:ContentPlaceHolder>
-
-                    <asp:ContentPlaceHolder ID="main" runat="server"></asp:ContentPlaceHolder>
-                </main>
-
-                 <footer>
-
-
-                        <hr />
-
-                        <div class="row">
-                            <div class="col-md-8">
-                                <Rock:Zone Name="Footer" runat="server" />
-                            </div>
-                            <div class="col-md-4">
-
-                            </div>
-                        </div>
-
-	            </footer>
-
-            </div>
-
-	    </div>
-
-        <%-- controls for scriptmanager and update panel --%>
-        <asp:ScriptManager ID="sManager" runat="server"/>
-        <asp:UpdateProgress id="updateProgress" runat="server" DisplayAfter="800">
-		        <ProgressTemplate>
-		            <div class="updateprogress-status">
-                        <div class="spinner">
-                          <div class="rect1"></div>
-                          <div class="rect2"></div>
-                          <div class="rect3"></div>
-                          <div class="rect4"></div>
-                          <div class="rect5"></div>
-                        </div>
-                    </div>
-                    <div class="updateprogress-bg modal-backdrop"></div>
-		        </ProgressTemplate>
-        </asp:UpdateProgress>
-
-    </form>
-
-    <script>
-        Sys.Application.add_load( function () {
-            $('.navbar-side > li').hover(function () {
-                if (!$('body').hasClass('zone-highlight')) {
-                    $(this).doTimeout('hover', 250, 'addClass', 'open');
-                }
-            }, function () {
-                $(this).doTimeout('hover', 250, 'removeClass', 'open');
-            });
-        });
-
-
-        $(document).ready(function () {
-
-            // enable sidbar sub menus
-            $('.navbar-side li').click(function () {
-                $('.navbar-side li').removeClass('open');
-                $(this).addClass('open');
-            });
-
-            $('#content-wrapper').click(function () {
-                $('.navbar-side li').removeClass('open');
-            });
-
-            // show/hide sidebar nav
-            $('.navbar-minimize').click(function () {
-                if ($('.navbar-static-side').is(":visible")) {
-                    $('body').addClass('navbar-side-close').removeClass('navbar-side-open');
-                } else {
-                    $('body').addClass('navbar-side-open').removeClass('navbar-side-close');
-                }
-            });
-        });
-    </script>
-
-</body>
-
-</html>
+﻿<%@ Master Language="C#" AutoEventWireup="true" Inherits="Rock.Web.UI.RockMasterPage" %>

```

Comment:
Not sure what's going on here with this Site.Master, but it's showing up as a changed file and now has ctrl-M at the end of every line.